### PR TITLE
Get Vanilla assets to load locally

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4916/
Test PASSed.

File: jujugui/static/gui/src/app/assets/css/base.scss [hunk 1]
```diff
@@ -52,6 +52,9 @@ $z-indexed-elements: search-results-list-block__tags,
     full-screen-mask,
     centered-column;
 
+// Asset path used by Vanilla to load fonts instead of asset server
+$asset-path: 'juju-ui/assets/';
```
Comment (hatched) [inline jujugui/static/gui/src/app/assets/css/base.scss]: Before this lands lets find out why this works...

Comment (anthonydillon) [inline jujugui/static/gui/src/app/assets/css/base.scss]: @hatched You were completely right. Had a doh moment. Fixed now.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4918/
Test PASSed.

**Comment by makyo** [comment]
:+1:

**Comment by anthonydillon** [comment]
Sorry missed QA steps for some reason:
- Load the GUI
- Check in the Network tab in dev tools that fonts are loading locally and not from assets.ubuntu.com.

**Comment by fabricematrat** [comment]
:+1:  QA :ok:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4924/
Test PASSed.

**Comment by anthonydillon** [comment]
Thanks :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**